content/patterns/disclosure/examples/disclosure-navigation-hybrid.html

```diff
@@ -18,185 +18,185 @@
+          <li><a href="disclosure-navigation.html">ディスクロージャーナビゲーションメニューの例</a></li>
+          <li><a href="disclosure-faq.html">よくある質問の回答用ディスクロージャー（表示/非表示）の例</a></li>
+          <li><a href="disclosure-image-description.html">画像説明用ディスクロージャー（表示/非表示）の例</a></li>
+          <li><a href="../../menubar/examples/menubar-navigation.html">ナビゲーションメニューバーの例</a></li>
         </ul>
-        <h2>Example Usage Options</h2>
+        <h2>使用例の選択肢</h2>
         <p>
-          This example demonstrates two different ways of implementing keyboard support.
-          Toggle between them with the following checkbox.
+          この例では、キーボードサポートを実装する 2 つの異なる方法を示しています。
+          次のチェックボックスでそれらを切り替えます。
         </p>
         <label for="arrow-behavior-switch">
           <input type="checkbox" id="arrow-behavior-switch" checked>
-          Include optional support for arrow keys, <kbd>Home</kbd>, and <kbd>End</kbd>
+          矢印キー、<kbd>Home</kbd>、及び <kbd>End</kbd> の任意のサポートを含める
         </label>
       </section>
 
       <section>
         <div class="example-header">
-          <h2 id="ex_label">Example</h2>
+          <h2 id="ex_label">例</h2>
         </div>
-        <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>
+        <div role="separator" id="ex_start_sep" aria-labelledby="ex_label ex_start_sep" aria-label="の開始"></div>
         <div id="ex1">
           <header class="sample-header">
-            <div class="title" id="id_website_title">Mythical University</div>
-            <div class="tagline">Using a disclosure + link pattern for navigation</div>
+            <div class="title" id="id_website_title">架空の大学</div>
+            <div class="tagline">ナビゲーションにディスクロージャー + リンクパターンを使用</div>
           </header>
-          <nav aria-label="Mythical University">
+          <nav aria-label="架空の大学">
             <ul id="exTest" class="disclosure-nav">
               <li>
-                <a href="#mythical-page-content" class="main-link">About</a>
-                <button type="button" aria-expanded="false" aria-controls="id_about_menu" aria-label="More About pages"></button>
+                <a href="#mythical-page-content" class="main-link">大学について</a>
+                <button type="button" aria-expanded="false" aria-controls="id_about_menu" aria-label="その他の「大学について」のページ"></button>
                 <ul id="id_about_menu" style="display: none">
                   <li>
-                    <a href="#mythical-page-content">Overview</a>
+                    <a href="#mythical-page-content">概要</a>
                   </li>
                   <li>
-                    <a href="#mythical-page-content">Administration</a>
+                    <a href="#mythical-page-content">管理</a>
                   </li>
                   <li>
-                    <a href="#mythical-page-content">Facts</a>
+                    <a href="#mythical-page-content">大学の概要</a>
```

Comment:
factsかぁ……
Overviewがある以上「概要」のままにするわけにはいかないので、かなり意訳ですがこれくらいでどうでしょう……？
```suggestion
                    <a href="#mythical-page-content">企業データ</a>
```